build.cake

```diff
@@ -145,6 +145,24 @@ Task("TestAtlasConnectivity")
     );
 });
 
+Task("TestGssapi")
+    .IsDependentOn("Build")
+    .DoesForEach(
+        GetFiles("./**/MongoDB.Driver.Tests.csproj"),
+        testProject => 
+{
```

Comment:
This indentation is weird. Why is the `{` not aligned with `testProject`?

Note: I realize it's also weird on another task. Perhaps we should fix it there also.

Reply:
Will add it in the next commit